src/OpenSearch.Net.VirtualizedCluster/VirtualCluster.cs

```diff
@@ -69,10 +69,10 @@ public VirtualCluster PublishAddress(string publishHost)
 			return this;
 		}
 
-		public VirtualCluster MasterEligible(params int[] ports)
```

Comment:
This would be a breaking change, I think you need both `Master` and `ClusterManager` versions, and tests to ensure we catch such. There's probably other similar instances.

Reply:
Good point. `VirtualizedCluster` is used by unit tests only, but users re-use code from it or from tests in their application.
Proposed fix - recover `MasterEligible` and call `ClusterManagerEligible` inside; add deprecation notice.

Reply:
@dblock,
please see fix in https://github.com/Bit-Quill/opensearch-net/pull/38.

Reply:
Looking good.

Reply:
Merged in [2d6cff5](https://github.com/opensearch-project/opensearch-net/pull/51/commits/2d6cff5b69af56004de813be87f4f182c346ffce).